webview/src/experiments/state/tableDataSlice.ts

```diff
@@ -49,7 +49,7 @@ export const tableDataSlice = createSlice({
     updateChanges: (state, action: PayloadAction<string[]>) => {
       state.changes = action.payload
     },
-    updateCliError: (state, action: PayloadAction<string | undefined>) => {
```

Comment:
Another possible solution is how we update our data as a whole. Right now we loop through existing keys. Maybe we check for keys that didn't exist in `data.data` and update missing keys to undefined?

Reply:
Switching undefined to null is the way I've fixed this previously.